src/clients/dotnet/samples/two-phase-many/TwoPhaseMany.csproj

```
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
```

```suggestion
    <RollForward>LatestMajor</RollForward>
  </PropertyGroup>
```

Let's add the `RollForward` tag to be compatible with newer runtimes (e.g., running using .net 7.0 even the project targeting 6.0). We can add the same tag for the other samples as well.

Without this tag:
```
❯ dotnet run
You must install or update .NET to run this application.

App: /home/batiati/tigerbeetle/src/clients/dotnet/samples/two-phase/bin/Debug/net6.0/TwoPhase
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '6.0.0' (x64)
.NET location: /usr/lib/dotnet

The following frameworks were found:
  7.0.5 at [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed
```

---

Ok! I'll add that to all of the sample projects then.